package.json

```diff
@@ -25,9 +25,10 @@
   "homepage": "https://github.com/odysseyscience/react-s3-uploader",
   "dependencies": {
     "aws-sdk": "2.x",
+    "koa-router": "^5.4.0",
```

Comment:
Koa would need to be in the server.js (presumably) already to work. You'll do

```
const app = koa()

app.use(s3routes(options))
```

No real way to not already have koa.